lib/sidekiq/antidote/inhibitor.rb

```diff
@@ -8,27 +8,31 @@ module Sidekiq
   module Antidote
     # Single poison inhibition rule: class qualifier + treatment action.
     class Inhibitor
-      TREATMENTS  = %w[skip kill].freeze
+      TREATMENTS  = %w[skip kill suspend].freeze
 
       # @return [String]
       attr_reader :id
 
-      # @return ["skip", "kill"]
+      # @return ["skip", "kill", "suspend"]
       attr_reader :treatment
 
       # @return [ClassQualifier]
       attr_reader :class_qualifier
 
+      # @return [String]
+      attr_reader :suspend_group
+
       # @param id [#to_s]
-      # @param treatment ["skip", "kill"]
+      # @param treatment ["skip", "kill", "suspend"]
       # @param class_qualifier [#to_s]
-      def initialize(id:, treatment:, class_qualifier:)
+      # @oaram suspend_group [#to_s]
+      def initialize(id:, treatment:, class_qualifier:, suspend_group: nil)
         @id              = -id.to_s
         @treatment       = -treatment.to_s
         @class_qualifier = ClassQualifier.new(class_qualifier.to_s)
+        @suspend_group   = suspend_group.to_s
```

Comment:
I don't think we need suspend group. We know the inhibitor ID at the time of treatment, so we can have list with `{REDIS_KEY}:suspend:{treatment.id}` to move the job to, and then upon inhibitor deletion, we will:

1. rename `{REDIS_KEY}:suspend:{treatment.id}` to `{REDIS_KEY}:release:{new_uuid}`
2. separate poller job will be responsible for draining those lists back to queues